tests/test_demography.py

```diff
@@ -6140,7 +6154,7 @@ def test_struct_metadata(self):
 
 class TestLineageMovementEvents:
     def test_as_lineage_movement_abstract(self):
-        e = msprime.LineageMovementEvent(1234)
+        e = msprime.demography.LineageMovementEvent(1234)
```

Comment:
I'm not following why the others get `demog_mod` and this one doesn't.

Reply:
No good reason, I'll change it.